addons/test_mail/tests/test_mail_message.py

```diff
@@ -252,6 +252,67 @@ def test_mail_message_values_fromto_reply_to_force_new(self):
         self.assertNotIn('-%d-' % self.alias_record.id, msg.message_id.split('@')[0])
 
 
+    def test_mail_message_starred_count(self):
```

Comment:
Try to order methods in a given test class alphabetically, to match the execution order